### PR TITLE
Fixed misplaced twitter button

### LINKED ISSUE #102: Twitter fab not placed properly
In the suggest fragment this button is not placed properly.

screenshot_20171220-131901

**Mobile and Android Version**
Samsung Galaxy J2(2016) Android Version : 6.0.1

**Would you like to work on this issue ?**
Yes.

### PR DESCRIPTION
**Pull request by mkmar01**
Fixes #102 

Twitter button now placed in proper position in the Suggest Fragment.

### CODE REVIEW

File: app/src/main/res/layout/fragment_suggest.xml [hunk 1]
```diff
@@ -55,6 +56,7 @@
 
     </android.support.v4.widget.SwipeRefreshLayout>
 
-    <include layout="@layout/fab_tweet_posting"/>
+    <include layout="@layout/fab_tweet_posting"
+        />
```
Comment (abishekvashok) [inline app/src/main/res/layout/fragment_suggest.xml]: Was this change needed?

Comment (mkmar01) [reply app/src/main/res/layout/fragment_suggest.xml]: should I change it again?

Comment (abishekvashok) [reply app/src/main/res/layout/fragment_suggest.xml]: Can you make it as before as this just removes intending.

Comment (mkmar01) [reply app/src/main/res/layout/fragment_suggest.xml]: yeah sure

**Comment by mkmar01** [comment]
@abishekvashok is it fine?

**Review by abishekvashok — APPROVED** [review]
Well done

**Review by singhpratyush — APPROVED** [review]

**Review by S2606 — APPROVED** [review]

**Review by kubami9 — APPROVED** [review]
